src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy

```diff
@@ -150,19 +149,30 @@ class ModuleDistribution extends DefaultTask
     @Input
     String getArtifactId()
     {
-        return subDirName.get()
+        return getSubDir()
     }
 
     String getArchiveName()
     {
         if (archiveName == null)
         {
-            var extraIdentifier = extraFileIdentifier != null ? extraFileIdentifier : "-" + project.name
+            var extraIdentifier = extraFileIdentifier != null ? extraFileIdentifier : "-" + getProjectName()
             archiveName = "${archivePrefix}${BuildUtils.getDistributionVersion(project)}" + extraIdentifier
         }
         return archiveName
     }
 
+    private String getSubDir()
+    {
+        return subDirName == null ? getProjectName() : subDirName;
+    }
+
+    private String getProjectName()
```

Comment:
I would rather not call this `getProjectName` since that has a particular meaning for Gradle. Perhaps `getDistName` or something like that

Reply:
Changed to `getDefaultName()`